tests/Unit/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Test_KerrSchild.cpp

```diff
@@ -155,6 +168,72 @@ void test_schwarzschild(const DataType& used_for_size) {
+  const std::array<double, 3> spin{{0.0, 0.0, 0.0}};
+  const std::array<double, 3> center{{0.0, 0.0, 0.0}};
+  gr::Solutions::KerrSchild solution(mass, spin, center);
+
+  // Setup grid
+  const size_t num_points_1d = 8;
+  const std::array<double, 3> lower_bound{{0.82, 1.24, 1.32}};
+  const std::array<double, 3> upper_bound{{0.8, 1.22, 1.30}};
+  const size_t SpatialDim = 3;
+  Mesh<SpatialDim> mesh{num_points_1d, Spectral::Basis::Legendre,
+                        Spectral::Quadrature::GaussLobatto};
+  const auto coord_map =
+      domain::make_coordinate_map<Frame::ElementLogical, FrameType>(Affine3D{
+          Affine{-1., 1., lower_bound[0], upper_bound[0]},
+          Affine{-1., 1., lower_bound[1], upper_bound[1]},
+          Affine{-1., 1., lower_bound[2], upper_bound[2]},
+      });
+  const size_t num_points_3d = num_points_1d * num_points_1d * num_points_1d;
+  // Setup coordinates
+  const auto x_logical = logical_coordinates(mesh);
+  const auto x = coord_map(x_logical);
+  // Arbitrary time for time-independent solution.
+  const double t = std::numeric_limits<double>::signaling_NaN();
+  // Evaluate analytic solution
+  const auto vars = solution.variables(
+      x, t,
+      typename gr::Solutions::KerrSchild::template tags<DataVector,
+                                                        FrameType>{});
+  const auto& spatial_metric =
+      get<gr::Tags::SpatialMetric<SpatialDim, FrameType>>(vars);
+  const auto& det_spatial_metric = determinant(spatial_metric);
+
+  // Compute actual analytical derivative of the determinant
+  const double null_vector_0 = -1.0;
+  gr::Solutions::KerrSchild::IntermediateComputer<DataVector, FrameType>
+      ks_computer(solution, x, null_vector_0);
+  gr::Solutions::KerrSchild::IntermediateVars<DataVector, FrameType> ks_cache(
+      solution, x);
+  auto H = make_with_value<Scalar<DataVector>>(
+      used_for_size, std::numeric_limits<double>::signaling_NaN());
+  ks_computer(make_not_null(&H), make_not_null(&ks_cache),
+              gr::Solutions::KerrSchild::internal_tags::H<DataVector>{});
+  const auto deriv_det_spatial_metric = ks_cache.get_var(
+      gr::Tags::DerivDetSpatialMetric<3, FrameType, DataVector>{});
+
+  // Compute expected numerical derivative of the determinant
+  using det_spatial_metric_tag = gr::Tags::DetSpatialMetric<DataVector>;
+  Variables<tmpl::list<det_spatial_metric_tag>> det_spatial_metric_var(
+      num_points_3d);
+  get<det_spatial_metric_tag>(det_spatial_metric_var) = det_spatial_metric;
+  const auto expected_deriv_det_spatial_metric_var =
+      partial_derivatives<tmpl::list<det_spatial_metric_tag>>(
+          det_spatial_metric_var, mesh, coord_map.inv_jacobian(x_logical));
+  const auto& expected_deriv_det_spatial_metric = get<
+      Tags::deriv<det_spatial_metric_tag, tmpl::size_t<SpatialDim>, FrameType>>(
+      expected_deriv_det_spatial_metric_var);
+
+  Approx approx = Approx::custom().epsilon(1e-11).scale(1.0);
```

Comment:
Note that even with 10, 11, or 12 points per dimension, it still does not improve the numerical approximation to be better than needing `1e-11`.